info.xml

```diff
@@ -14,12 +14,15 @@
     <url desc="Support">https://github.com/reflexive-communications/civalpa/issues</url>
     <url desc="Licensing">http://www.gnu.org/licenses/agpl-3.0.html</url>
   </urls>
-  <releaseDate>2021-03-10</releaseDate>
-  <version>0.2.0</version>
+  <releaseDate>2021-04-07</releaseDate>
+  <version>0.3.0</version>
   <develStage>alpha</develStage>
   <compatibility>
     <ver>5.0</ver>
   </compatibility>
+  <requires>
+    <ext>rc-base</ext>
```

Comment:
What is your opinion about specifing here a minimun version that is needed? Like: `<ext version="^0.4">` (composer style)
I know it is not respected, that could be just an extra info. Also I can't see now too much use cases, but maybe in the future...

Reply:
According to the [doc](https://docs.civicrm.org/dev/en/latest/extensions/info-xml/#ext) the version dependency is not implemented. If they do it, the version tag could be used for some other reason as you suggested it here. In this case it could cause issues.

If we really want to administrate the version of the dependencies, we could put it into the readme file.

Reply:
OK that's fair.